ImageItem/ImageItem.js

```diff
@@ -88,20 +96,32 @@ const ImageItemBase = kind({
 		className: 'imageItem'
 	},
 
-	render: ({children, css, src, ...rest}) => {
-		const [Component, marqueeProps] = children ? [MarqueeImageItem, {
+	computed: {
+		className: ({captionOverImage, styler}) => styler.append({
+			captionOverImage
+		})
+	},
+
+	render: ({captionOverImage, children, css,  src, ...rest}) => {
+		const imageItemWithOverlayCaption = (children && captionOverImage) ? [UiImageItem, children] : [UiImageItem, null];
```

Comment:
If children && captionOverImage, then `children` is passed to `Component` twice. 
One for `{...marqueeProps}`, the other for `{children}`.

I think that the conditions get complex. Let's make the code more comprehensible.

Reply:
I couldn't understand why not `[UiImageItem, null]` but `[UiImageItem, children]`?

Reply:
You are right, this works if I use [UiImageItem, null].